synapse/storage/databases/main/events.py

```diff
@@ -159,7 +175,7 @@ async def _persist_events_and_state_updates(
         #
         # Note: Multiple instances of this function cannot be in flight at
         # the same time for the same room.
-        if backfilled:
+        if use_negative_stream_ordering:
```

Comment:
Behavior argument replacement: `use_negative_stream_ordering`

Reply:
> Behavior argument replacement: `use_negative_stream_ordering`

sorry, what does this mean?

Reply:
It's just the root place that I originally replaced and defines the behavior